README.md

```diff
@@ -25,8 +25,29 @@ To train the model using CCLE data, execute the following command. This is will
 
 The outputs and the logs get written to the output_dir specified in the swnet_ccle_model.txt.
 
-
-
+## Running the model outside container
+### Installation
+---
+Install the requirements (listed in environment.yaml). We're using Anaconda to install the environment:
+```
+conda env create -f environment.yaml
+conda activate swnet
+pip install numpy==1.17
+```
+Install CANDLE library:
+```
+pip install git+https://github.com/ECP-CANDLE/candle_lib.git@develop
+```
+### Modification
+Modify line 14 in train.sh such that:
+```
+CANDLE_MODEL=SWnet_CCLE_baseline_pytorch.py
```

Comment:
Thanks, but this should not be necessary. The train.sh script has to be written so it always works. Either add the script to the search path and make it executable or add this to the script:
```
### Set env if CANDLE_MODEL is not in same directory as this script
IMPROVE_MODEL_DIR=${IMPROVE_MODEL_DIR:-$( dirname -- "$0" )}
CANDLE_MODEL=${IMPROVE_MODEL_DIR}/${CANDLE_MODEL}
```